src/tests/java/htsjdk/variant/variantcontext/filter/JavascriptVariantFilterTest.java

```diff
@@ -0,0 +1,36 @@
+package htsjdk.variant.variantcontext.filter;
+import htsjdk.variant.vcf.VCFFileReader;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.io.File;
+
+/**
+ * @author Pierre Lindenbaum PhD Institut du Thorax - INSERM - Nantes - France
+ */
+
+public class JavascriptVariantFilterTest {
+	final File testDir = new File("testdata/htsjdk/variant");
+	
+    @DataProvider
+    public Object [][] jsData() {
+        return new Object[][] {
+                {"ILLUMINA.wex.broad_phase2_baseline.20111114.both.exome.genotypes.1000.vcf", "variantFilter01.js"},
+                {"ILLUMINA.wex.broad_phase2_baseline.20111114.both.exome.genotypes.1000.vcf", "variantFilter02.js"},
+        };
+    }
+
+	@Test(dataProvider = "jsData")
+    public void testJavascriptFilters(final String vcfFile, final String javascriptFile)
+    	{
+    	final File vcfInput = new File(testDir, vcfFile);
+    	final File jsInput = new File(testDir, javascriptFile);
+        final VCFFileReader vcfReader = new VCFFileReader(vcfInput, false);
+    	final JavascriptVariantFilter filter = new JavascriptVariantFilter(jsInput, vcfReader.getFileHeader());
+    	final FilteringIterator iter = new FilteringIterator(vcfReader.iterator(), filter);
+    	while(iter.hasNext()) iter.next();
```

Comment:
same here, could you verify that the "right" amount of records passes through the filter?